uint32 degree
int32 velocity
---
bool success
